data/strings/strings.txt

```diff
@@ -23057,25 +22963,24 @@
     lt = įspėti jei yra greičio viršijimo rizika
     mr = वेगमर्यादा ओलांडण्याचा धोका असल्यास चेतावणी द्या
     nb = Advarsel hvis det er fare for å overskride fartsgrensen
-    nl = Waarschuw als er een risico bestaat dat de snelheidslimiet wordt overschreden
+    nl = Waarschuw bij overschrijden snelheidslimiet
     pl = Ostrzegaj jeśli istnieje ryzyko przekroczenia ograniczenia prędkości
-    pt = Avisar se houver risco de exceder o limite de velocidade
-    pt-BR = Avisar se houver risco de ultrapassar o limite de velocidade
-    ro = Avertizează dacă există riscul depășirii limitei de viteză
-    ru = Предупреждать если есть риск превышения скоростного лимита
+    pt = Avisar se exceder limite de velocidade
+    ro = Avertizează la depășirea limitei de viteză
+    ru = Предупреждать при превышении скорости
     sk = Upozornenie ak existuje riziko prekročenia rýchlostného limitu
```

Comment:
Many longer translations can be shortened. What about this?

```
    en = To warn of speeding
    af = Waarsku teen spoed
    ar = التحذير من السرعة
    az = Sürətlə bağlı xəbərdarlıq edin
    be = Папярэджваць аб перавышэнні хуткасці
    bg = Да ви предупреди за превишена скорост
    ca = Avisar sobre l'excés de velocitat
    cs = Varování před překročením rychlosti
    da = At advare dig om hastighedsoverskridelser
    de = Um dich vor Geschwindigkeitsüberschreitungen zu warnen
    el = Για να σας προειδοποιήσει για υπερβολική ταχύτητα
    en-GB = To warn you of speeding
    es = Para advertirte del exceso de velocidad
    es-MX = Advertir sobre exceso de velocidad
    et = Et hoiatada teid kiiruse ületamise eest
    eu = Abiarazi abiaduraz
    fa = هشدار در مورد سرعت غیر مجاز
    fi = Varoittaa sinua ylinopeudesta
    fr = Pour t'avertir d'un excès de vitesse
    he = התריעו על מהירות מופרזת
    hi = तेज़ गति से गाड़ी चलाने के बारे में चेतावनी दें
    hu = Figyelmeztetni a gyorshajtásra
    id = Untuk memperingatkan Anda agar tidak ngebut
    it = Per avvisarti di un eccesso di velocità
    ja = スピード違反を警告する
    ko = 과속 경고
    lt = Įspėti apie greičio viršijimą
    mr = वेगाबद्दल चेतावणी द्या
    nb = For å advare deg mot fartsovertredelser
    nl = Om je te waarschuwen voor te hard rijden
    pl = Aby ostrzec Cię przed przekroczeniem prędkości
    pt = Para te avisar de excesso de velocidade
    pt-BR = Para avisar você sobre excesso de velocidade
    ro = Pentru a vă avertiza în caz de exces de viteză
    ru = Предупреждать о превышении скорости
    sk = Varovanie pred prekročením rýchlosti
    sv = För att varna dig för fortkörning
    sw = Onya kuhusu mwendo kasi
    th = เตือนเรื่องการขับรถเร็ว
    tr = Sizi hız konusunda uyarmak için
    uk = Попереджати про перевищення швидкості
    vi = Cảnh báo về việc chạy quá tốc độ
    zh-Hans = 警告您超速行驶
    zh-Hant = 警告超速
```

Reply:
The Romanian translation you sugest is longer than the alternative.

Reply:
Of course, it's better to take shorter ones. I'm not insisting on replacing everything, these are autogenerated by DeepL and Google.

Reply:
> The Romanian translation you suggest is longer than the alternative.

I'm keeping the translation
    `ro = "Avertizează la depășirea limitei de viteză"`

Reply:
> Many longer translations can be shortened. What about this?

I've updated those translations that _significantly_ shorten the strings...